dartagnan/src/main/antlr4/LitmusC.g4

```diff
@@ -72,13 +76,13 @@ re locals [IOpBin op, String mo]
         | AtomicDecReturn        LPar address = re RPar {$op = IOpBin.SUB; $mo = Linux.MO_MB;}
         | AtomicDecReturnRelaxed LPar address = re RPar {$op = IOpBin.SUB; $mo = Linux.MO_RELAXED;}
         | AtomicDecReturnAcquire LPar address = re RPar {$op = IOpBin.SUB; $mo = Linux.MO_ACQUIRE;}
-        | AtomicDecReturnRelease LPar address = re RPar {$op = IOpBin.SUB; $mo = Linux.MO_RELEASE;})                  # reAtomicOpReturn
-    
-    |   ( C11AtomicAdd LPar address = re Comma value = re Comma c11Mo RPar {$op = IOpBin.ADD;}
-        | C11AtomicSub LPar address = re Comma value = re Comma c11Mo RPar {$op = IOpBin.SUB;}
-        | C11AtomicOr LPar address = re Comma value = re Comma c11Mo RPar {$op = IOpBin.OR;}
-        | C11AtomicXor LPar address = re Comma value = re Comma c11Mo RPar {$op = IOpBin.XOR;}
-        | C11AtomicAnd LPar address = re Comma value = re Comma c11Mo RPar {$op = IOpBin.AND;})                         # C11AtomicOp
+        | AtomicDecReturnRelease LPar address = re RPar {$op = IOpBin.SUB; $mo = Linux.MO_RELEASE;})                                    # reAtomicOpReturn
+
+    |   ( C11AtomicAddExplicit LPar address = re Comma value = re Comma c11Mo RPar {$op = IOpBin.ADD;}
+        | C11AtomicSubExplicit LPar address = re Comma value = re Comma c11Mo RPar {$op = IOpBin.SUB;}
+        | C11AtomicOrExplicit LPar address = re Comma value = re Comma c11Mo RPar {$op = IOpBin.OR;}
+        | C11AtomicXorExplicit LPar address = re Comma value = re Comma c11Mo RPar {$op = IOpBin.XOR;}
+        | C11AtomicAndExplicit LPar address = re Comma value = re Comma c11Mo RPar {$op = IOpBin.AND;})                                 # C11AtomicOp
```

Comment:
You are missing all non-explicit versions

Reply:
Added